kalite/topic_tools/__init__.py

```diff
@@ -81,26 +82,32 @@ def recurse_nodes(node):
             else:
                 # By default this is very charitable, assuming if something has not been annotated
                 # it is available - needs to be updated for exercises.
-                if content_cache.get(node.get("id"), {}).get("languages", True):
+                if content_cache.get(node.get("id"), {}).get("available", True):
                     node["available"] = True
 
-        recurse_nodes(TOPICS[channel])
+            # Translate everything for good measure
+            translation.activate(i18n.lcode_to_django_lang(language))
```

Comment:
Interesting, maybe it's time to wrap the `translation.activate/deactivate` calls into a decorator/context manager. Could you make one real quick?